components/Intro.tsx

```diff
@@ -45,40 +74,184 @@ const Intro = () => {
   }, [admin, isFirstRender]);
 
   return (
-    <section className="intro-section">
-      <div className="container">
-        <div className="intro-content">
-          <div className="intro-avatar">
-            <div className="image-avatar">
+    <Box
+      sx={(theme) => ({
+        py: 8.75,
+        background: theme.palette.mode === "light" ? "#f8fafc" : "#070809",
+        color: theme.palette.mode === "light" ? "#707070" : "#f2f5f7",
+      })}
+    >
+      <Box className="container">
+        {/* check, do we need to rewrite this className? */}
```

Comment:
This className has 4 responsive and using at 7 different component, what do you think about this?